### ISSUE TITLE
index.php link in home board portal with ultimate SEO

### ISSUE THREAD

**Issue by ceds59** [phpBBSEO/usu#108]
Hello
i installed ultimate seo extension, but i use Board3 Portal and the link "Forum index" redirect on the portal
i want this link redirect on index.php
Please help for this solution

**Comment by juarol** [comment]
This would be better asked over at board3 official support forum. The portal historically has always controlled this behavior.

**Comment by ceds59** [comment]
No, cause the problèm appears when i installed ultimate seo for phpbb3.1.x
.htaccess particularity ?
Other file to modify ?
Anterior version there was phpbb_seo_class to modify for this problem 
Before no problem 
Please help

Cédric

**Comment by ceds59** [comment]
No support ???
extension abandonned ?????

**Comment by Alxston** [comment]
No, no support apparently. Not abandoned just starving to near death.

**Comment by ceds59** [comment]
What'interest to create an extension for abandonned it ???   

Cédric DA SILVA

**Comment by Leinad4Mind** [comment]
I guess you need to add this line:
     # FORUM PORTAL
     RewriteRule ^portal\.html$ /forum/portal.php [QSA,L,NC]

before:
     # FORUM INDEX
     RewriteRule ^forum\.html$ /forum/index.php [QSA,L,NC]

**Comment by ceds59** [comment]
hello

on http://forum-malinois.net/app.php/portal
index.php is OK on Index du FOrum

on http://forum-malinois.net/
index.php not avalaible in Index du FOrum

same after modification

----- Mail original -----
De: "Daniel Mota" notifications@github.com
À: "phpBBSEO/usu" usu@noreply.github.com
Cc: "ceds59" cedsp1@free.fr
Envoyé: Samedi 27 Février 2016 04:10:55
Objet: Re: [usu] index.php link in home board portal with ultimate SEO (#108)

I guess you need to add this line: 

# FORUM PORTAL

RewriteRule ^portal.html$ /forum/portal.php [QSA,L,NC] 

before: 

# FORUM INDEX

RewriteRule ^forum.html$ /forum/index.php [QSA,L,NC]

**Comment by Leinad4Mind** [comment]
@ceds59 Did you solve this issue? If so, tell us how so we can closed this.
